metron-platform/metron-enrichment/src/test/java/org/apache/metron/enrichment/integration/EnrichmentIntegrationTest.java

```diff
@@ -196,6 +201,15 @@ public static void validateAll(List<Map<String, Object>> docs) {
     }
   }
 
+  protected void validateErrors(List<byte[]> inputMessages, KafkaWithZKComponent kafkaComponent) {
```

Comment:
I changed getProcessor() to attempt to read the ERROR and INVALID streams from Kafka already in a past PR to improve integration test error handling.  If they need to read from the bolts that you have, maybe you can change it there, and then pass the  results into here and not have to read them